common/app/views/fragments/atoms/structureddata/filmReview.scala.html

```diff
@@ -2,14 +2,15 @@
 @import org.joda.time.DateTime
 @import play.api.libs.json.Json
 @import play.api.libs.json.JsArray
+@import model.Tag
 
-@(review: model.content.ReviewAtom, entity: Film)(implicit request: RequestHeader)
+@(review: model.content.ReviewAtom, entity: Film, contributors: Seq[Tag])(implicit request: RequestHeader)
 
 <script type="application/ld+json">
 {
   "@@context":"http://schema.org",
   "@@type":"Review",
-  "author": @fragments.atoms.structureddata.person(review.data.reviewer),
+  "author": @fragments.atoms.structureddata.contributor(review.data.reviewer, contributors),
```

Comment:
contributor doesn't pertain only to recipes so doesn't belong in a recipe folder.